releasenotes/notes/fix-addition-parameterised-paulisumop-e9b859bca1e58ddd.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Addition between :class:`~qiskit.opflow.PauliSumOp`s that have
+    :class:`~qiskit.circuit.ParameterExpression` coefficients raises `QiskitError`.
+    Now we support the addition.
```

Comment:
```suggestion
    Addition between :class:`~qiskit.opflow.PauliSumOp`s that had
    :class:`~qiskit.circuit.ParameterExpression` coefficients raised `QiskitError`.
    Now we support the addition.
```

Reply:
I changed it slightly as 'raises' still felt like the present tense where it still might do it.